packages/flutter_tools/lib/src/commands/update_packages.dart

```diff
@@ -30,63 +30,63 @@ import '../runner/flutter_command.dart';
 const Map<String, String> kManuallyPinnedDependencies = <String, String>{
   // Add pinned packages here. Please leave a comment explaining why.
   'flutter_gallery_assets': '1.0.2', // Tests depend on the exact version.
-  'flutter_template_images': '4.0.0', // Must always exactly match flutter_tools template.
+  'flutter_template_images':
+      '4.0.0', // Must always exactly match flutter_tools template.
```

Comment:
extraneous newline

Reply:
in fact it looks like a whole bunch of formatting changes were made to this file (it's a pretty good illustration of why we don't use dart format, some of the changes are really much worse than what we have now!).

Reply:
We will have to agree to disagree about the merits of automatic formatting, but I did revert the change :).